docs/operating-eck/troubleshooting/take-eck-dump.asciidoc

```diff
@@ -0,0 +1,125 @@
+:page_id: take-eck-dump
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Take ECK Dump
+
+
+Steps to take ECK dump:
+
+[float]
+== Prepare
+
+To download the ECK dump shell file, run:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+curl -L -O https://raw.githubusercontent.com/elastic/cloud-on-k8s/master/support/diagnostics/eck-dump.sh
```

Comment:
Nit: I would combine the two commands for simplicity:
```
curl -L -O https://raw.githubusercontent.com/elastic/cloud-on-k8s/master/support/diagnostics/eck-dump.sh && chmod +x eck-dump.sh
```

Reply:
Thanks for the comment here! For simplicity, I am very happy to do it as you advised!
But sorry I am not very sure how to delete the "Change mode to executable" section in my later part.
May I trouble you to help with removing the corresponding statement please?